models/bamf_pet_ct_lung_tumor/config/default.yml

```diff
@@ -0,0 +1,47 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: default configuration for Bamf NNUnet Lung and FDG-avid lesions in the lung from FDG PET/CT scans (dicom to dicom)
+
+execute:
+- FileStructureImporter
+- module: NiftiConverter
+  in_datas: dicom:mod=ct
+- module: NiftiConverter
+  in_datas: dicom:mod=pt
+  engine: dcm2niix
```

Comment:
I used single engine initially and observed few things.
- plastimatch and dcm2niix produced different results with same pt dicom input
`root@5f7eff9a8e96:/app# plastimatch stats pt_plastimatch.nii.gz  pt_dcm2niix.nii.gz`
`MIN -6089.604980 AVE 143.495453 MAX 6089.418945 NONZERO 6200765 NUMVOX 11520000`
`MIN 0.000000 AVE 149.425613 MAX 58740.855469 NONZERO 6200765 NUMVOX 11520000`
- dcm2niix engine was failing with below error for ct input
`Converting /app/data/input_data/pat1/ct/5aa84950-844f-478c-b09a-cdac4a923dda.dcm`
`Dims 512 512 5 1 1`
`Warning: Interslice distance varies in this volume (incompatible with NIfTI format).`
`Distance frError: Too many NIFTI images with the name /app/data/imported_instances/pat1/nifti/ct_e1`

so I had to declare `NiftiConverter` with different engine here

Reply:
yes. The training pipeline used [SitkNiftiConverter](https://github.com/bamf-health/aimi-lung-pet-ct/blob/main/app/src/converter_utils.py#L48) for PET/CT. Using `NiftiConverter` with `dcm2niix` engine gave similar output with inference. But with `plastimatch` engine, the tumor segmentation seems to performs badly.

I also noticed that `dcm2niix` engine fails for few ct scans. so changing `NiftiConverter` with `dcm2niix` for PET and `NiftiConverter` with `plastimatch` engine for CT produced similar results and worked fine without errors.

Reply:
The second error was observed on CT scans.

Reply:
Could we then use `dcm2niix` for PET and CT?

Reply:
`mhubai/base` image uses `dcm2niix==1.0.20181125` which causes error for `NiftiConverter` . One example being `s3://idc-open-data/aafd2a22-6236-4eff-b541-1dfd8de923e8/*`

I have updated `dcm2niix==1.0.20220715` to fix this issue in current model image

Reply:
What kind of error does the most-recent version cause? Is it a general error or only causing issues with some nifti files (thus, it might be a problem with the file itself)?

Reply:
Yes. The files had warnings around `Interslice Distance` and `Missing Images`
[error.txt](https://github.com/user-attachments/files/16700076/error.txt)

`dcm2niix==1.0.20220715` seems to preprocess and handle them without errors.

Reply:
I believe the underlying issue in dcm2niix has been fixed. The problem is that the base mhub image has an old version of dcm2niix, from 2018, we need a 2022 or newer version of dcm2niix

Reply:
Okay, apparently the installation of dcm2niix via apt loads a very old version. I've updated our base image with the newer version, tested it with all models that use dcm2niix as an engine, and released it. I'm glad we had this discussion because I think this is a great improvement for our base image. Could you please `docker pull mhubai/base:latest`, remove the dcm2niix install lines and see if everything works as expected now? Please remove this in all other PRs as well.

Reply:
`mhubai/base` has dcm2niix==1.0.20220715. I will update this in other PRs

Reply:
Great! @vanossj @jithenece thank you both, this is a great finding and helped improving the base image towards an up-to-date and robust version.